cli/README.md

```diff
@@ -31,18 +31,34 @@ The site build bundles `metadata.properties` along with the archetype catalog in
 This file is hosted on helidon.io. For example:
 https://helidon.io/cli-data/2.2.2/cli-data.zip
 
+### Before versions 3.x
```

Comment:
Can you make a small paragraph in the beginning that mentions this issue and PR and describes the overall change.

Then after we can have two top-level sections `3.0.4 or less` and `After 3.0.4`.